commcare_connect/opportunity/forms.py

```diff
@@ -616,6 +616,8 @@ class Meta:
         help_texts = {
             "duration": "Form completion duration threshold. (minutes)",
             "location": "Check form distance to other form submission. (metres)",
```

Comment:
I would mention that these are the minimum duration and distance between forms. maybe something like

> Minimum time to complete form (minutes)

and 

> Minimum distance between form locations (metres)

Reply:
Thanks. Added. https://github.com/dimagi/commcare-connect/pull/301/commits/d7d5d30d3f689bc69747adba8928b3705815a2fc